packages/main/src/Link.js

```diff
@@ -72,9 +72,9 @@ const metadata = {
 		 * @defaultvalue "Default"
 		 * @public
 		 */
-		type: {
-			type: LinkType,
-			defaultValue: LinkType.Default,
+		design: {
```

Comment:
Look at the jsdoc above "Defines the <code>ui5-link</code> type." Replace "type" with "design"